src/main/steps/task-list/template.njk

```diff
@@ -193,7 +193,7 @@
             </li>
             <li class="app-task-list__item">
                 <span class="app-task-list__task-name">
-                    <a href="#" class="govuk-link" aria-describedby="eligibility-completed">
+                    <a href="{{urls.BIRTH_FATHER_IS_NAME_ON_CERTIFICATE}}" class="govuk-link" aria-describedby="eligibility-completed">
```

Comment:
urls is custom object we pass from task-list content, you might need to add entry in it
https://github.com/hmcts/adoption-web/blob/49656f812231830f659ffe2a3eeff3d723e767d9/src/main/steps/task-list/content.ts#L111

https://github.com/hmcts/adoption-web/blob/49656f812231830f659ffe2a3eeff3d723e767d9/src/main/steps/task-list/content.ts#L214